public/video-ui/src/components/Videos/AdvancedActions.js

```diff
@@ -6,22 +6,31 @@ class AdvancedActions extends React.Component {
     permissions = getStore().getState().config.permissions;
     showActions = this.permissions.deleteAtom;
 
+    state = { deleteDoubleCheck: false }
+
     renderDelete() {
         if(!this.permissions.deleteAtom) {
             return false;
         }
 
-        const doDelete = () => {
-            const result = prompt("Enter the atom ID to confirm deletion (it can be copied from the URL)");
+        const disabled = this.props.usage.length > 0;
+        const deleteMsg = this.state.deleteDoubleCheck ? "Confirm delete from database" : "Delete from database";
 
-            if(result === this.props.video.id) {
+        const doDelete = () => {
+            if(this.state.deleteDoubleCheck) {
                 this.props.videoActions.deleteVideo(this.props.video);
+            } else {
+                this.setState({ deleteDoubleCheck: true });
             }
         };
 
         return <li className="action-list__item">
-            <button className="btn label__expired action-list__button" onClick={doDelete}>DELETE</button>
-            <span className="right">Usages will not be replaced and the video will remain on YouTube</span>
+            <button className="btn label__expired action-list__button" onClick={doDelete} disabled={disabled}>
+                {deleteMsg}
+            </button>
```

Comment:
Good point. I've updated the message on the left to explain why the button is disabled:

```
All usages of the atom must be removed before deletion
```